### DISCUSSION TITLE
iPadminiの第6世代で配信できない

### DISCUSSION THREAD

**Discussion by rtmp-dev** [HaishinKit/HaishinKit.swift#1108] [Q&A]
お世話になってます。

現在HaishinKitを使用していますが、
以下の環境で映像配信ができなくなってしまいました。

現在、iPadminiの第6世代に対応していないなどありますか？
また、もし対応される予定などあれば対応時期もお伺いできますでしょうか？

【機種】iPadmini 第６世代
【OS】iPadOS 16.1.1
【HaishinKit Ver】1.3.0
【配信先】Youtube
【使っている機能】ScreenCaptureSession
　※以下のドキュメント記載のAPIを使用しています。
　https://shogo4405.github.io/HaishinKit.swift/Classes/ScreenCaptureSession.html#/s:10HaishinKit20ScreenCaptureSessionC06viewToD0ACSo6UIViewC_tcfc
　https://shogo4405.github.io/HaishinKit.swift/Classes/NetStream.html#/s:10HaishinKit9NetStreamC12attachScreen_03useF4SizeyAA25CaptureSessionConvertible_pSg_SbtF

**Comment by shogo4405** [comment]
iPad mini 第6世代を所持していないため。わからないです。
また、実機の購入予定がないため対応も難しいです。

**Reply by rtmp-dev** [reply]
ご返信ありがとうございました。本件につきまして承知いたしました。
